docker/Dockerfile

```diff
@@ -24,5 +24,13 @@ RUN cd /tiered-storage-for-apache-kafka \
     && tar -xf tiered-storage-for-apache-kafka-${_VERSION}.tgz --strip-components=1 \
     && rm tiered-storage-for-apache-kafka-${_VERSION}.tgz
 
+# Installing JMX exporter agent
+ENV JMX_EXPORTER_VERSION=0.18.0
```

Comment:
This probably should be ARG instead. Otherwise, it's possible to build with one version and then to run with another (without success).